### ISSUE TITLE
iOS - Disable VR mode screen goes black

### ISSUE THREAD

**Issue by justinwasilenko** [googlesamples/cardboard-unity#196]
In my game there is an option to disable VR mode to play outside of the Cardboard if you want to. 

For iOS users they are reporting when disabling VR mode the screen goes black and they can't see anything. They are then uninstalling the app and reinstalling to get it work again.

Is disabling VR mode no longer supported in v0.6 of the plugin on iOS?

**Comment by smdol** [comment]
No, it's a bug.  We just have to figure out where.  Are you using Metal or OpenGL?  Do you have Direct Render on or off?

**Comment by eljaydub** [comment]
In Unity's Player Settings our Rendering Path is set to 'Forward' and we are using 'Auto Graphics API'. I assume that will then use Metal, but in any case those are the settings used by the build where users are experiencing the bug.

**Comment by smdol** [comment]
We have spotted a bug with Unity + Metal + Direct Render = off that is vaguely similar sounding.  We don't know if the issue is ours, Unity's, or Metal's.  Is there a way you test setting the Graphics API to OpenGL?

**Comment by eljaydub** [comment]
We were able to reproduce the issue consistently with a build from Unity 5.3.2 and the settings I listed above.

I just updated to Unity 5.3.4 and tested it, here are the results:

- Using only Metal I could switch between VR and non-VR modes ok (ie no black screen)
- Using only OpenGL ES 2 I could switch between VR and non-VR modes ok
- Using 'Auto Graphics API' I could switch between VR and non-VR modes ok

So it appears something changed from Unity 5.3.2 to 5.3.4 that has fixed the error. If we still get complaints from users when the new iOS version goes out we will report here and investigate further.

Thanks!
